evalml/tests/automl_tests/test_automl.py

```diff
@@ -176,7 +180,7 @@ def test_pipeline_fit_raises(mock_fit, X_y_binary, caplog):
     mock_fit.side_effect = Exception(msg)
     X, y = X_y_binary
 
-    automl = AutoMLSearch(problem_type='binary', max_iterations=1)
+    automl = AutoMLSearch(problem_type='binary', max_iterations=1, train_best_pipeline=False)
```

Comment:
Don't train the best pipeline, since this test mocks the `pipeline.fit()` method and causes it to raise an exception, which we don't want to raise while fitting the best pipeline.

Reply:
Got it. This could be a good comment to add.